release/internal/pinnedversion/pinnedversion.go

```diff
@@ -300,3 +299,19 @@ func RetrieveVersions(outputDir string) (version.Versions, error) {
 
 	return version.NewHashreleaseVersions(version.New(pinnedVersion.Title), pinnedVersion.TigeraOperator.Version), nil
 }
+
+// normalizeComponent normalizes the component image name.
+// It checks if the image is in the registry.ImageMap and replaces it with the mapped value.
+// If the image is not found in the map, it sets it to the name of the component.
+// The image is also stripped of the calico namespace prefix if it exists.
+func normalizeComponent(name string, c registry.Component) registry.Component {
+	img := registry.ImageMap[c.Image]
```

Comment:
@radTuti  @coutinhop  The previous code on the left uses name as the key instead of c.Image,
img := registry.ImageMap[name] 
In the new code, we are fetch the img with c.Image key which results in an empty img value. As a result, the assignment ends up using the default name "flexvol" which is incorrect here.